src/types/severa/workHour/workHour.ts

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
Check indentation settings and fix indentation, these should all be indented the same amount but they are not.